admin/modules/forum/attachments.php

```diff
@@ -668,20 +668,23 @@ function() {
 		}
 
 		// LESS THAN or GREATER THAN
-		if($mybb->input['dateuploaded'] && $mybb->request_method == "post")
-		{
-			$mybb->input['dateuploaded'] = TIME_NOW-$mybb->input['dateuploaded']*60*60*24;
+		$direction_fields = array(
+			"dateuploaded" => $mybb->get_input('dateuploaded', 1),
+			"filesize"     => $mybb->get_input('filesize', 1),
+			"downloads"    => $mybb->get_input('downloads', 1),
+		);
+
+		if ($mybb->input['dateuploaded'] && $mybb->request_method == "post") {
```

Comment:
This and a similar line below don't comply with the standards yet - `{` should be in a new line and no space between `if` and `(`.